### PR TITLE
11 BLB cards

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/jackdaw_savior.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Jackdaw Savior
+ManaCost:2 W
+Types:Creature Bird Cleric
+PT:3/1
+K:Flying
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self,Creature.Other+YouCtrl+withFlying | Execute$ TrigReturn | TriggerDescription$ Whenever CARDNAME or another creature you control with flying dies, return another target creature card with lesser mana value from your graveyard to the battlefield.
+SVar:TrigReturn:DB$ ChangeZone | ValidTgts$ Creature.CMCLTX+YouOwn+NotDefinedTriggeredCard | TgtPrompt$ Select another target creature card with lesser mana value in your graveyard | Origin$ Graveyard | Destination$ Battlefield
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jackdaw_savior.txt:7]: **CMC** property does not exist

File: forge-gui/res/cardsfolder/upcoming/lunar_convocation.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ManaCost:W B
+Types:Enchantment
+T:Mode$ Phase | Phase$ End of Turn | TriggerZones$ Battlefield | ValidPlayer$ You | CheckSVar$ X | SVarCompare$ GE1 | Execute$ TrigLoseLife | TriggerDescription$ At the beginning of your end step, if you gained life this turn, each opponent loses 1 life.
+SVar:TrigLoseLife:DB$ LoseLife | Defined$ Player.Opponent | LifeAmount$ 1
+T:Mode$ Phase | Phase$ End of Turn | TriggerZones$ Battlefield | ValidPlayer$ You | CheckSVar$ Y | SVarCompare$ GE1 | Execute$ TrigToken | TriggerDescription$ At the beginning of your end step, if you gained and lost life this turn, create a 1/1 black Bat creature token with flying.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ b_1_1_bat_flying | TokenOwner$ You
+A:AB$ Draw | Cost$ 1 B PayLife<2> | NumCards$ 1 | SpellDescription$ Draw a card.
+SVar:X:Count$LifeYouGainedThisTurn
+SVar:Y:PlayerCountPropertyYou$LifeLostThisTurn/Times.X
+AI:RemoveDeck:Random
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/lunar_convocation.txt:11]: deckneeds lifegain

File: forge-gui/res/cardsfolder/upcoming/splash_portal.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Splash Portal
+ManaCost:U
+Types:Sorcery
+A:SP$ ChangeZone | ValidTgts$ Creature.YouCtrl | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select target creature you control | RememberTargets$ True | SubAbility$ DBReturn | StackDescription$ Exile {c:Targeted}, | SpellDescription$ Exile target creature you control, then return that card to the battlefield under its owner's control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/splash_portal.txt:4]: wrong description

File: forge-gui/res/cardsfolder/upcoming/splash_portal.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Splash Portal
+ManaCost:U
+Types:Sorcery
+A:SP$ ChangeZone | ValidTgts$ Creature.YouCtrl | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select target creature you control | RememberTargets$ True | SubAbility$ DBReturn | StackDescription$ Exile {c:Targeted}, | SpellDescription$ Exile target creature you control, then return that card to the battlefield under its owner's control.
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Destination$ Battlefield | RememberChanged$ True | ForgetOtherRemembered$ True | SubAbility$ DBDraw | StackDescription$ then return it to the battlefield under its owner's control.
+SVar:TrigDraw:DB$ Draw | Defined$ You | ConditionDefined$ Remembered | ConditionPresent$ Bird,Frog,Otter,Rat | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/splash_portal.txt:7]: add deckhints type

File: forge-gui/res/cardsfolder/upcoming/sinister_monolith.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Sinister Monolith
+ManaCost:3 B
+Types:Artifact
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDrain | TriggerDescription$ At the beginning of combat on your turn, each opponent loses 1 life and you gain 1 life.
+SVar:TrigDrain:DB$ LoseLife | Defined$ Player.Opponent | LifeAmount$ 1 | SubAbility$ DBGainLife
+SVar:DBGainLife:DB$ GainLife | Defined$ You | LifeAmount$ 1
+A:AB$ Draw | Cost$ T PayLife<2> Sac<1/CARDNAME> | NumCards$ 2 | SorcerySpeed$ True | SpellDescription$ Draw two cards. Activate only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sinister_monolith.txt:7]: deckhas lifegain

File: forge-gui/res/cardsfolder/upcoming/hazels_nocturne.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+Name:Hazel's Nocturne
+ManaCost:3 B
+Types:Instant
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Hand | TargetMin$ 0 | TargetMax$ 2 | TgtPrompt$ Choose target creature card in your graveyard | ValidTgts$ Creature.YouOwn | SubAbility$ DBDrain | SpellDescription$ Return up to two target creature cards from your graveyard to your hand. Each opponent loses 2 life and you gain 2 life.
+SVar:DBDrain:DB$ LoseLife | Defined$ Player.Opponent | LifeAmount$ 2 | SubAbility$ DBGainLife
+SVar:DBGainLife:DB$ GainLife | Defined$ You | LifeAmount$ 2
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/hazels_nocturne.txt:6]: deckhas lifegain

File: forge-gui/res/cardsfolder/upcoming/splash_portal.txt (hunk 4 above)
Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/splash_portal.txt:7]: OK, this should be all comments addressed.

**Review by tool4ever — APPROVED** [review]

**Pull request merged**